emmaa/db/manager.py

```diff
@@ -287,6 +336,23 @@ def get_users(self, query, model_id):
             users = [q for q, in q.all()]
         return users
 
+    def update_subscription(self, user_id, query_hash, new_sub_status):
+        with self.get_session() as sess:
+            # Get current subscription
+            (curr_sub,), = sess.query(UserQuery.subscription).filter(
+                UserQuery.user_id == user_id,
+                UserQuery.query_hash == query_hash
+            )
+            # If requested subscription status is not current one, change it
+            if new_sub_status is not curr_sub:
+                sess.query(UserQuery).filter(
+                    UserQuery.user_id == user_id,
+                    UserQuery.query_hash == query_hash
+                ).update({'subscription': new_sub_status})
+                sess.commit()
+                logger.info(f'Updated subscription status to '
+                            f'{new_sub_status} for query {query_hash}')
+
```

Comment:
This is for future proofing. Some time soon, we'll want to let people un-subscribe their subscribed queries and then we want to have a way of only doing that and nothing else.

Reply:
Sure, but I still think the code could be _much_ simpler. In fact, if you want them to be able to unsubscribe, that would not change the code. In the external for-loop you could just do:
```
if subscribe is not None:
    userquery.subscription = subscribe
```
And for now that would be the same, almost exactly:
```
if subscribe:
    userquery.subscription = subscribe
````

Reply:
I imagine we will have a button next to a subscribed query that let's the user un subscribe. This will result in a POST request that triggers the un-subscription.